src/test/java/pageobject/GoogleComputingEnginePage.java

```diff
@@ -0,0 +1,29 @@
+package pageobject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class GoogleComputingEnginePage {
+
+    private WebDriver driver;
+
+    @FindBy(id = "input_53")
+    private WebElement numberOfInstances;
```

Comment:
org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"id","selector":"input_53"}

invalid locator